Packs/SophosXGFirewall/Integrations/sophos_firewall.py

```diff
@@ -0,0 +1,2020 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+import json
+import requests
+import urllib3
+from typing import Union, Optional, Callable
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+VERSION = '1702.1'
```

Comment:
```suggestion
API_VERSION = '1702.1'
```

Reply:
should not it be configurable with an integration parameter?

Reply:
No, the API versions are very different, so the endpoints aren't the same on other versions. Just wanted to make sure the version is clear and accessible in the code.
I use the variable only once in the code, and I can remove it, but I think it's more friendly the way it is. What do you think?